pandas/plotting/_matplotlib/misc.py

```diff
@@ -210,7 +210,7 @@ def f(t):
             # appropriately. Take a copy of amplitudes as otherwise numpy
             # deletes the element from amplitudes itself.
             coeffs = np.delete(np.copy(amplitudes), 0)
-            coeffs.resize(int((coeffs.size + 1) / 2), 2)
+            coeffs.resize(int((coeffs.size + 1) / 2), 2, refcheck=False)
```

Comment:
out of curiosity, how is this causing a warning?

Reply:
Can't reproduce anymore. Reverting.